src/Avatar/utilis.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { stringToHslaColor } from './utils';
+
+const randomString = Math.random().toString(36).substring(2, 5);
+
+describe('stringToHslaColor', () => {
+  it.each(['test', 'test'])(
+    'is a pure function returning the same hsla for the same string',
+    (testString) => {
+      expect(stringToHslaColor(testString)).toStrictEqual(
+        'hsla(58, 78%, 68%, 0.9)',
+      );
+    },
+  );
+
+  it('returns various hsla colors for various inputs', () => {
+    const testStrings = ['t', 'te', 'tes', 'test', randomString];
```

Comment:
Does not hold true if `randomString` is exactly `'test'`.

Reply:
I think the test is still valid and good. what would you do about it? Leave it out at all?